<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarParkingManager</class>
 <widget class="QMainWindow" name="CarParkingManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Zarządzanie Parkingiem Samochodowym</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- Przycisk ENTRYBUTTON -->
   <widget class="QPushButton" name="ENTRYBUTTON">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>350</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>Zarejestruj Wjazd</string>
    </property>
    <property name="styleSheet">
     <string>font-size: 26px; background-color: #4CAF50; color: white; border-radius: 10px;</string>
    </property>
   </widget>
   <!-- Przycisk EXITBUTTON -->
   <widget class="QPushButton" name="EXITBUTTON">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>350</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>Zarejestruj Wyjazd</string>
    </property>
    <property name="styleSheet">
     <string>font-size: 26px; background-color: #f44336; color: white; border-radius: 10px;</string>
    </property>
   </widget>
   <!-- Przycisk loadImageButton do wczytywania obrazu samochodu -->
   <widget class="QPushButton" name="loadImageButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>350</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>Wczytaj Obraz</string>
    </property>
    <property name="styleSheet">
     <string>font-size: 26px; background-color: #2196F3; color: white; border-radius: 10px;</string>
    </property>
   </widget>
   <!-- Pole do podglądu obrazu imagePreview -->
   <widget class="QLabel" name="imagePreview">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>750</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>border: 2px solid #cccccc; border-radius: 10px; background-color: #f0f0f0;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <!-- Etykieta label_2 do wyświetlania informacji -->
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>1150</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="styleSheet">
     <string>font-size: 24px; color: #333333; font-weight: bold; padding-left: 10px;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <!-- Pole wyboru numeru rejestracyjnego -->
   <widget class="QComboBox" name="comboBoxRejestracja">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>670</y>
      <width>350</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>font-size: 22px; padding: 5px; border: 1px solid #cccccc; border-radius: 5px;</string>
    </property>
   </widget>
   <!-- Przyciski miejsc parkingowych -->
   <widget class="QWidget" name="miejscaParkingoweWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>770</y>
      <width>750</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="ukladMiejscParkingowych">
     <!-- Dodanie 5 miejsc parkingowych -->
     <item row="0" column="0">
      <widget class="QPushButton" name="s1">
       <property name="text">
        <string>S1</string>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string>font-size: 18px; background-color: #40FF50; border-radius: 10px; min-width: 130px; min-height: 60px; max-width: 300px; max-height: 100px; padding: 10px;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="s2">
       <property name="text">
        <string>S2</string>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string>font-size: 18px; background-color: #40FF50; border-radius: 10px; min-width: 130px; min-height: 60px; max-width: 300px; max-height: 100px; padding: 10px;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="s3">
       <property name="text">
        <string>S3</string>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string>font-size: 18px; background-color: #40FF50; border-radius: 10px; min-width: 130px; min-height: 60px; max-width: 300px; max-height: 100px; padding: 10px;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="s4">
       <property name="text">
        <string>S4</string>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string>font-size: 18px; background-color: #40FF50; border-radius: 10px; min-width: 130px; min-height: 60px; max-width: 300px; max-height: 100px; padding: 10px;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="s5">
       <property name="text">
        <string>S5</string>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string>font-size: 18px; background-color: #40FF50; border-radius: 10px; min-width: 130px; min-height: 60px; max-width: 300px; max-height: 100px; padding: 10px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>